src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -181,6 +181,7 @@ def load_arguments(self, _):
                    '--node-vm-size', '-s'], completer=get_vm_size_completion_list)
         c.argument('nodepool_name', type=str, default='nodepool1',
                    help='Node pool name, up to 12 alphanumeric characters', validator=validate_nodepool_name)
+        c.argument('os_sku', type=str, options_list=['--os-sku'], completer=get_ossku_completion_list)
```

Comment:
The value `--os-sku` in `options_list` is the default value of the parameter, so you don't need to add it
```suggestion
        c.argument('os_sku', type=str, completer=get_ossku_completion_list)
```

Reply:
Thanks, I didn't know those were the default values! I've committed the suggestions.